modules/shared/partials/acid-transactions.adoc

```diff
@@ -250,48 +258,37 @@ However, there are situations that inevitably created failed, or 'lost' transact
 
 This requires an asynchronous cleanup task, described in this section.
 
-Creating the `Transactions` object spawns a background cleanup task, whose job it is to periodically scan for expired transactions and clean them up.
+The transactions cleanup task periodically scans for expired transactions and cleans them up.
```

Comment:
I'm concerned this makes it sound like a process is involved, rather than something in-client.  It's an important distinction as we need the user to be aware that if their app isn't running, cleanup isn't either.